lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -647,7 +647,10 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 		}
 
 		if ( in_array( 'styles', $types, true ) ) {
+			$stylesheet .= $this->get_root_layout_rules( static::ROOT_BLOCK_SELECTOR, $style_nodes[ $root_block_key ] );
 			$stylesheet .= $this->get_block_classes( $style_nodes );
+
+			$root_block_key = array_search( static::ROOT_BLOCK_SELECTOR, array_column( $style_nodes, 'selector' ) );
```

Comment:
This line needs to be moved to before the `$stylesheet .= $this->get_root_layout_rules` line above, otherwise the layout rules aren't output.

There should always be an element for the root block selector, but I was wondering if we should also add an `if ( false !== $root_block_key )` check before calling `get_root_layout_rules` in case a match isn't found?